packages/jest-worker/src/workers/__tests__/WorkerEdgeCases.test.ts

```diff
@@ -188,7 +189,7 @@ describe.each([
       expect(startPid).toBeGreaterThanOrEqual(0);
       expect(worker.state).toEqual(WorkerStates.OK);
 
-      expect(orderOfEvents).toMatchObject(['ok']);
+      expect(orderOfEvents).toEqual(['ok']);
```

Comment:
Felt like `toEqual` is better idea here. `toMatchObject` gave a type error. Looking at this code, it works with arrays, but typings allow only `Record`. Should I fix typings?

Reply:
>Should I fix typings?

Yeah, that sounds reasonable.

That said, I agree `toEqual` is better in this case 🙂